src/content-switcher/content-switcher.component.ts

```diff
@@ -31,7 +34,10 @@ export class ContentSwitcher implements AfterViewInit {
 
 	@ContentChildren(ContentSwitcherOption) options: QueryList<ContentSwitcherOption>;
 
+	constructor(private elementRef: ElementRef) {}
```

Comment:
Use `protected` so components are extendible.
```suggestion
	constructor(protected elementRef: ElementRef) {}
```